feat: document ttl for api keys (#1899)
ory/docs: docs/concepts/personal-access-token.mdx

1. Go to <ConsoleLink route="project.developers" />.
2. Click **Create new API key**
3. Enter the API Key name

> 4. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.

Reviewer: ```suggestion
4. Select an **expiration** date. Note that it is not recommended to create API keys without an expiry date.
```

> 4. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.

Reviewer: ```suggestion
4. Select an **expiration** date. It is recommended to always create API keys with an expiry date.
```

> 4. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.

Reviewer 2: I disagree with this recommendation. Expiring API keys are a recipe for outages 😆 . Also typo in `recommended`

Author: Typo fixed by @vinckr :D.

Do you have an alternative suggestion regarding our wording here? Long-living access tokens are also a security threat, but too short-lived keys are a threat for expiring too soon, creating outages.

Reviewer: Maybe just describe it like you did. 

Finding the best time to live for an access token is always a tradeoff between security and usability. For maximum security use a short lived access token, but consider that expiring API keys could lead to outages or other issues. For maximum usability use a long-lived access token, but consider that it can be a security risk if the token leaks. 

(personally I dont understand why 1 month/1 day access token have different security profile - if it leaks you are screwed either way no?)

1. Go to <ConsoleLink route="project.developers" />.
2. Click **Create new API key**
3. Enter the API Key name
4. Select an **expiration** date.
5. Click **Create**
6. Copy the created API Key

> Finding the best expiration date for an API token is always a tradeoff between security and usability. For maximum security use a

Reviewer 2: ```suggestion
Finding the best expiration date for an API key is always a tradeoff between security and usability. Rotating tokens is generally a good practice, however it comes at the cost of increased maintenance -- automatic or manual. If your API keys expire before you renew them, your service might degrade or become unavailable.
```

Follow these steps to create API keys for your workspace:

> [//] #1. Go to the [Ory Console](https://console.ory.sh/). 2. Go to **Workspace settings** through the workspace menu in the top

Reviewer 3: this format is wrong, list does not work(render)

2. Go to **Workspace settings** through the workspace menu in the top left corner.
3. Select **API keys** in the sidebar.
4. Click the **+** icon in the **API Keys** section.
5. Enter the API Key name

> 6. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.

Reviewer: ```suggestion
6. Select an **expiration** date. Note that it is not recommended to create API keys without an expiry date.
```

> 6. Select an **expiration** date. Note that it is not recommeded to create API keys without an expiry date.

Reviewer: ```suggestion
6. Select an **expiration** date. It is recommended to always create API keys with an expiry date.
```
